pihole

```diff
@@ -421,7 +421,8 @@ Debugging Options:
                         Add '-a' to automatically upload the log to tricorder.pi-hole.net
   -f, flush           Flush the Pi-hole log
   -r, reconfigure     Reconfigure or Repair Pi-hole subsystems
-  -t, tail            View the live output of the Pi-hole log
+  -t, tail            View the live output of the Pi-hole log.
+                        Add an argument to be filter the log
```

Comment:
```suggestion
  -t, tail [arg]      View the live output of the Pi-hole log.
                      Add the optional argument to filter the log
                      (regular expressions are supported)
```

Reply:
I was mainly concerned about the extra `be` in the original text but then I though we should also add where to add the argument and that it supports regular expressions.